src/Atlas.cc

```diff
@@ -357,6 +361,9 @@ void Atlas::PostLoad()
 void Atlas::SaveAtlas(string pathSaveFileName, string strVocabularyName, string strVocabularyChecksum) {
     lock_guard<mutex> lock(mMutexAtlas);
     this->PreSave();
+    
+    pathSaveFileName = pathSaveFileName.append(".osa");
```

Comment:
Were we previously saving maps without any extension?

Reply:
It was built into the map saving thread we had set up. Was planning to move it here to match functionality of System::SaveAtlas

Reply:
kk actually going to remove this, better to leave any filename modifiers in the server